android/src/main/java/com/pdftron/reactnative/utils/Constants.java

```diff
@@ -89,6 +89,7 @@ public final class Constants {
     public static final String TOOL_BUTTON_EDIT = "editToolButton";
     public static final String TOOL_BUTTON_ADD_PAGE = "addPageButton";
     public static final String TOOL_BUTTON_INSERT_PAGE = "insertPageButton";
+    public static final String TOOL_BUTTON_COUNT = "countToolButton";
```

Comment:
don't think we need this one, customer can just use the tool version

Reply:
removed